doc-content/enterprise-only/openshift/rhpam711-trial-ephemeral-ref.adoc

```diff
@@ -116,7 +116,7 @@ https://cloud.google.com/container-engine/docs/services/[container-engine docume
 A route is a way to expose a service by giving it an externally reachable hostname such as `www.example.com`. A defined route and the endpoints
 identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
 of a route name, service selector, and (optionally) security configuration. See the
-https://access.redhat.com/documentation/en-us/openshift_enterprise/3.11/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
+https://access.redhat.com/documentation/en-us/openshift_enterprise/3.2/html/architecture/core-concepts#architecture-core-concepts-routes[Openshift documentation] for more information.
```

Comment:
[link] The link returns 404. The latest version of OCP 3 is 3.11.

s/3.2/3.11

"https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/architecture/core-concepts#architecture-core-concepts-routes"